hyperspy/tests/io/test_hdf5.py

```diff
@@ -355,12 +355,13 @@ def test_oom_loading(self):
         assert self.shape == s.data.shape
         assert isinstance(s.data, da.Array)
         assert s._lazy
+        s.close_file()
 
     def teardown_method(self, method):
         gc.collect()        # Make sure any memmaps are closed first!
         try:
```

Comment:
What is the reason to let the tests pass if we can't remove the file? If an error is raised, it is most likely because something is wrong and we need to fix it?